test/e2e/mcd_test.go

```diff
@@ -533,3 +534,46 @@ func TestCustomPool(t *testing.T) {
 	err = waitForPoolComplete(t, cs, "infra", renderedConfig)
 	require.Nil(t, err)
 }
+
+func TestIgn3Cfg(t *testing.T) {
+	cs := framework.NewClientSet("")
+
+	// create a dummy MC with an sshKey for user Core
+	mcName := fmt.Sprintf("ign3cfg-worker-%s", uuid.NewUUID())
+	mcadd := &mcfgv1.MachineConfig{}
+	mcadd.ObjectMeta = metav1.ObjectMeta{
+		Name:   mcName,
+		Labels: mcLabelForWorkers(),
+	}
+	// create a new MC that adds a valid user & ssh key
+	Ign3Config := ign3types.Config{}
+	tempUser := ign3types.PasswdUser{Name: "core", SSHAuthorizedKeys: []ign3types.SSHAuthorizedKey{"1234_test_ign3"}}
+	Ign3Config.Passwd.Users = append(Ign3Config.Passwd.Users, tempUser)
+	Ign3Config.Ignition.Version = "3.0.0"
```

Comment:
https://github.com/openshift/machine-config-operator/blob/fcos/pkg/controller/common/helpers.go#L159-L175

Reply:
yeah i considered doing this but as we don't support ign3 yet I didnt want to create a public function creating them.